src/apps/external_executor/executor_code_id.h

```diff
@@ -50,6 +50,12 @@ inline std::pair<grpc_status, std::string> verification_error(
       return std::make_pair(
         GRPC_STATUS_UNAUTHENTICATED,
         "Quote report data does not contain node's public key hash");
+    case ccf::QuoteVerificationResult::FailedHostDataDigestNotFound:
+      return std::make_pair(
+        GRPC_STATUS_UNAUTHENTICATED, "Quote does not contain host data");
+    case ccf::QuoteVerificationResult::FailedInvalidHostData:
+      return std::make_pair(
+        GRPC_STATUS_UNAUTHENTICATED, "Quote host data is not authorised");
```

Comment:
These changes look unrelated?

Reply:
Yeah they are, but I figured they were worth getting in anyway

Reply:
I think the way this is implemented will change in the near future, but for now this is ok.